lib/utils.py

```diff
@@ -410,3 +410,12 @@ def find_tags(filename):
             else:
                 break
     return tags
+
+
+def prepend_path(p):
+    """ Add an absolute path into PATH (at start) if it is not already there.
+    """
+    p = os.path.abspath(p)
+    if p in os.environ['PATH'].split(os.pathsep):
```

Comment:
Looks like we should check that the provided path is the first item on the list. If I had the `PATH` variable already modified manually, say, I added `/path/to/test-run/lib/luatest/bin` after `/usr/bin`, then in a few days I installed luatest into the system (now it is in `/usr/bin`), this wouldn’t work as expected.

Reply:
I need the following information from you to ensure that we understood each other.

1. Whether you mean the case of testing of tarantool (so locally built tarantool will be added to the PATH) or testing of a module (so system tarantool will be used)?
2. Whether you mean using `./test/test-run.py`, using `./test/test-run.py --env` + `luatest` or both?
3. What is expected result and what is actual result in given conditions?

Reply:
Imagine the situation that I want to use `test-run` to run some tests via its `luatest` executable. I don't have `luatest` installed in my system (`luatest` executable is not present in `/usr/bin/`) and I want to use just filename `luatest` without specifying full executable path. To do it I change my `PATH` so that `PATH=/usr/bin:/path/to/test-run/luatest:<…>`. As you can see `/usr/bin` goes first. So I use `luatest` as usual and everything is fine. 

Later for some reason I install `luatest` into my system (now `luatest` is in `/usr/bin/`). Thus `luatest` from `/usr/bin/` is used for my purposes and this is fine to me.

Then I want to run `test-run` to execute some luatest tests. When I do it, the `luatest` executable form `/usr/bin/` will be used because `/usr/bin` goes first in `PATH`. You just check that test-run's luatest executable path is on the list and return. So it is not enough.

Reply:
Actually, this patch will be very helpful for users who are going to use built-in test-run's luatest. So I am not going to block this PR and ask solve my case. Let's resolve this in the scope of a separate issue.